marked-element.html

```diff
@@ -200,36 +200,47 @@
         return;
       }
 
-      this.markdown = this._unindent(this._markdownElement.textContent);
+      if (this._markdownElement.textContent.trim() !== '') {
+        this.markdown = this._unindent(this._markdownElement.textContent);
+      }
 
       if (this._markdownElement.src) {
         this._request(this._markdownElement.src);
-      } else {
-        var observer = new MutationObserver(this._onScriptAttributeChanged
-            .bind(this));
-        observer.observe(this._markdownElement, { attributes: true })
       }
+
+      this._observer = new MutationObserver(this._onScriptAttributeChanged
```

Comment:
The current tests are failing in Firefox for 1.x because the `src` binding is happening between `ready` and `attached` so not getting picked up by the `MutationObserver`. Working to resolve this.